py4DSTEM/process/diffraction/crystal_ACOM.py

```diff
@@ -27,6 +36,7 @@ def orientation_plan(
     fiber_axis=None,
     fiber_angles=None,
     figsize: Union[list, tuple, np.ndarray] = (6, 6),
+    CUDA: bool = True,
```

Comment:
Don't make this True by default!

Reply:
Good point!  I got too excited about adding CUDA support.